### PR TITLE
PLANET-5275: Running accessibility report with pa11y

### PR DESCRIPTION
**Pull request by lithrel**
Ref: https://jira.greenpeace.org/browse/PLANET-5275
Ref: https://github.com/greenpeace/planet4/issues/73

Requires: https://github.com/greenpeace/planet4-base-fork/pull/91 (pa11y configuration)  
Requires: https://github.com/greenpeace/planet4-docker-compose/pull/52 (targets for pa11y installation and run)

A report generated by circle-ci is visible there https://9211-80401279-gh.circle-artifacts.com/0/planet4-docker-compose/artifacts/pa11y/index.html

### CODE REVIEW

File: .circleci/config.yml
Comment (comzeradd) [inline .circleci/config.yml:127]: Do you think, this can be broken in into a separate job? (eg. a11y-tests). That way it can run in parallel, but most importantly it would allow us to merge this and have the tests run without blocking PRs until we fix all the issues reported.

Comment (lithrel) [reply .circleci/config.yml]: So I've made it run in another job, and [it seems to work](https://app.circleci.com/pipelines/github/greenpeace/planet4-master-theme/1606/workflows/f19ad42a-2edb-44fc-b852-ae4797c260a5), but because I still want to have a report of warnings (and maybe notices), I'm doing something not pretty with jq and an exit code, don't know if there's a better way.  
What I'm wondering is, how can I configure this job so that it fails without blocking a PR merge ?

Comment (comzeradd) [reply .circleci/config.yml]: I just adjusted repo settings, so instead of requiring the whole `main` workflow to pass it would require only the 3 jobs we want. We will still see a11y job check, but it won't block the merge button.

**Review by comzeradd — APPROVED** [review]
Looks nice! Extra :pancakes: for making abstracting re-usable config parts.

**Comment by comzeradd** [comment]
Oh, and don't forget to update branches before merge :exclamation:

**Pull request merged**
